python/pylibcudf/pylibcudf/transform.pyx

```diff
@@ -47,16 +52,18 @@ cpdef tuple[gpumemoryview, int] nans_to_nulls(Column input):
     """
     cdef pair[unique_ptr[device_buffer], size_type] c_result
 
+    stream = _get_stream(stream)
+
     with nogil:
-        c_result = cpp_transform.nans_to_nulls(input.view())
+        c_result = cpp_transform.nans_to_nulls(input.view(), stream.view())
 
     return (
         gpumemoryview(DeviceBuffer.c_from_unique_ptr(move(c_result.first))),
```

Comment:
Does it make sense to pass `stream` to `c_from_unique_ptr` also?

Reply:
Good call. Yes, we should do that. I need to make sure we do this everywhere I think. I'll take note on https://github.com/rapidsai/cudf/issues/15163 to verify that everywhere.